contracts/core/RevenuePath.sol

```diff
@@ -178,7 +170,7 @@ contract RevenuePath is Ownable, Initializable, ReentrancyGuard {
      * @param walletCount Length of wallet list
      * @param distributionCount Length of distribution list
      */
-    error WalletAndDistributionCountMismatch(uint256 walletCount, uint256 distributionCount);
+    error WalletAndDistrbtionCtMismatch(uint256 walletCount, uint256 distributionCount);
```

Comment:
While renaming these just keep in mind:
 1. The message conveys whatever the error does and explains it . We don't need to save words for that.
 2. In tests, we get these replaced.